### PR TITLE
add flash animation

### PR DESCRIPTION
**Pull request by tavvy**
🐿 v2.5.16

### CODE REVIEW

File: myft/ui/myft-buttons/main.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
+@mixin myftHeaderIconFlash {
+
+	@-webkit-keyframes myft-flash {
```
Comment (remybach) [inline myft/ui/myft-buttons/main.scss]: I think we use `autoprefixer` so you don't need to include any of these.

File: myft/ui/myft-buttons/main.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+			@include oGridRespondTo('M') {
+				width: 88px;
+			}
+
+			@include oGridRespondTo('L') {
+				width: 100px;
+			}
+
+			-webkit-animation: myft-flash linear 1.2s;
```
Comment (remybach) [inline myft/ui/myft-buttons/main.scss]: ... or these...

File: myft/ui/myft-buttons/main.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		25% { opacity: 1; }
+		50% { opacity: 1; }
+	}
+
+	@keyframes myft-flash {
+		25% { opacity: 1; }
+		50% { opacity: 1; }
+	}
+
+	@-webkit-keyframes myfy-flash-white {
```
Comment (pixelandpage) [inline myft/ui/myft-buttons/main.scss]: typo? myfy --> myft

File: myft/ui/myft-buttons/main.scss (hunk 1 above)
Comment (tavvy) [reply myft/ui/myft-buttons/main.scss]: nice, good catch!

Comment (tavvy) [reply myft/ui/myft-buttons/main.scss]: we do indeed, i verified on the built file 👍

**Pull request merged**